### DISCUSSION TITLE
命令自动升级提示错误

### DISCUSSION THREAD

**Discussion by xiamuguizhi** [ArtalkJS/Artalk#100] [Q&A] [answered]
版本：Artalk (v2.1.0/5d5bc6d)

``` bash
bash-5.1# artalk-go upgrade
FATA[0000] 请检查配置文件，并设置一个 app_key (任意字符串) 用于数据加密 
```

**Comment by qwqcode** [comment, marked as answer]
https://github.com/ArtalkJS/ArtalkGo/commit/996b762d5baf99492d37f94d02c16e3b87cd0d9f 已修复，重新 docker pull 创建容器就 OK

**Reply by xiamuguizhi** [reply]
![image](https://user-images.githubusercontent.com/19650297/163737699-c2f7f4cc-2ab8-48aa-af77-672dadca103d.png)

输入命令 显示正常！！是否在官网 添加 关于输入命名的自动升级 ？ 我还是去版本介绍找的 `artalk-go upgrade`

...

接下来我们使用该命令进入一个已经在运行的容器

` sudo docker ps  ` 查看在运行的容器 获取 容器ID

```bash

CONTAINER ID   IMAGE              COMMAND                  CREATED         STATUS         PORTS                     NAMES
c83d12c5928c   artalk/artalk-go   "/usr/bin/artalk-go …"   5 minutes ago   Up 4 minutes   0.0.0.0:8080->23366/tcp   artalk-go

```

在输入一下命令  容器 ID ：c83d12c5928c    进入容器bash

` sudo docker exec -it 容器ID /bin/bash  ` 

进入容器后 输入  `artalk-go upgrade` 完成自动检测升级。

ps：宝塔用户 可直接使用 Docker管理器 点击终端 输入  `artalk-go upgrade` 完成自动检测升级 即可！

**Reply by qwqcode** [reply]
OK！有空就加上这段说明
